### PR TITLE
Add ArtiCAD Ltd to the users page 

### PR DESCRIPTION
**Pull request by Benjy1979**

### CODE REVIEW

File: data/user/articad.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+url: 'https://www.articad.com'
+image: articad.png
+name: Articad Ltd Design to Help You Sell
```
Comment (amyrlam) [inline data/user/articad.md]:
```suggestion
name: Articad Ltd
```

File: data/user/articad.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+added: 2019-03-25T19:57:46.494Z
+---
+This is where you fill in how you use Ember
```
Comment (amyrlam) [inline data/user/articad.md]:
```suggestion
Design to Help You Sell
```

or another description of your choice re: how you use Ember

Comment (amyrlam) [reply data/user/articad.md]: https://github.com/ember-learn/ember-website/blob/master/data/user/intercom.md they just wrote `Intercom's main web application is built with Ember.js`

Comment (Benjy1979) [reply data/user/articad.md]:
```suggestion
ArtiCAD uses ember.js for it's room planner and styler application. 
```

Comment (Benjy1979) [reply data/user/articad.md]: Thanks for your pointers

**Pull request merged**